.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -46,6 +46,18 @@ As minimally and precisely as possible. Keep in mind we do not have access to yo
 
 --->
 
+**If this is a UI bug, please provide a screenshot of the bug or a link to a youtube video of the bug in action**
```

Comment:
Anything we leave in bold will likely be left in place by users, so this line should probably be in the comment.